src/cli.ts

```diff
@@ -59,7 +75,7 @@ function startCommandProcess(args: CLIArguments) {
 }
 
 /**
- * Initiates a webpack build for the extending project
+ * Initiates a webpack build for the extending project.
```

Comment:
What's the standard. This makes it inconsistent with other comments in this file